stable/kong/README.md

```diff
@@ -93,6 +93,13 @@ and their default values.
 | resources                      | Pod resource requests & limits                                                   | `{}`                |
 | tolerations                    | List of node taints to tolerate                                                  | `[]`                |
 
+### Admin/Proxy listener override
+
+If you specify `env.admin_listen` or `env.proxy_listen`, this chart will use
+the value provided by you as opposed to constructing a listen variable
```

Comment:
Can we add the bit about how the containerPort still needs to match with the listener the user provides in order to render services and other templates correctly?

Reply:
Updated